.eslintrc.yml

```diff
@@ -0,0 +1,12 @@
+env:
```

Comment:
this file just sets up the rules for javascript linting now that I am using unit tests and es6 etc.